### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Core Identity, Sign in Service, Sign out all devices

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#68805]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

- I gave a lot of feedback in the meeting but overall this looks good!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Must:** If you end up using the red buttons, just make sure you stick to one of the [design system reds](https://design.va.gov/foundation/color-palette). If possible darker is probably going to be better, but I think `secondary` and `secondary-dark` should be fine.
- **Must:** For each session, the buttons to sign out need to be distinguishable --- so if a screen reader user tabs to one of those buttons, it's clear what session is going to be ended by activating that button. If possible, I always like visible text. If that's not practical for design reasons ("Sign out of your Windows 11 session" is admittedly a little wordy), then an `aria-label` on the button would be sufficient, eg. `<button aria-label="Sign out device - Windows 11">Sign out device</button>` or something similar.
- **Must:** Echoing Erin in #68798, I don't know that there's a lot to be gained by putting the sign out button inside an additional info. But if you do go that route, the additional info triggers need to be distinguishable along the same lines as the buttons, eg. "View session information for Windows 11 device" or similar.
- **Must:** Green is a tricky color for color contrast and readability. I don't have a specific recommendation for the "Current Session" label, but I would recommend trying some combination of:
   - Bigger text size
   - Black text with the green icon, optionally with some other visual indicator to group them together (background color, outline, etc)
- **Must:** It sounds like this is already on your radar, but definitely have the buttons trigger a confirmation modal. An accidental click with no confirmation might lead to a user logged out, on the homepage, and very confused about how they got there.
- **Must:** When signing out of a session besides the current session (eg. desktop user signing out of phone), there needs to be some kind of inline success message --- and that success message should receive keyboard focus. That focus management will ensure the message is announced to screen reader users (so they know the thing they just did worked), and will ensure that keyboard-only users stay in a relatively consistent location on the page.
- **Consider:** I don't have specific content recommendations, but I do think a little more context and plain language could be helpful. "Windows 11 - Current Session" requires users to (1) know their operating system, and (2) understand what "current session" means. Even if it adds to the wordiness, I think there's value in being a little more descriptive.

**Comment by claytonzook** [comment]
Must
- [ ] We decided to use the standard blue color button for now
- [ ] Noted, we'll pass this along to our engineering team when they start coding
- [ ] Based on the overwhelming feedback, we decided to pull out the buttons, so no longer hidden on mobile!
- [ ] We decided to go black text with green icon. I also noticed this text was a touch below standard smallest size in the design system, so bumped it up to meet the DS
- [ ] 100% on the modal
- [ ] this success messaging makes sense - we'll work with our engineers to make sure we put this together
